packages/e2e/local-network/scripts/install.sh

```diff
@@ -18,7 +18,7 @@ trap clean EXIT
 
 VERSION="1.35.0"
 LINUX_BUILD="16159630"
-MACOS_BUILD="16159630"
```

Comment:
Was this a copy/paste error, or an issue with the build?

Reply:
copy/past error